packages/core-components/src/components/ProgressBars/Gauge.tsx

```diff
@@ -104,19 +119,40 @@ export const getProgressColor: GaugePropsGetColor = ({
  */
 
 export function Gauge(props: GaugeProps) {
+  const hoverRef = useRef<HTMLDivElement>(null);
   const { getColor = getProgressColor } = props;
   const classes = useStyles(props);
   const { palette } = useTheme<BackstageTheme>();
-  const { value, fractional, inverse, unit, max } = {
+  const { value, fractional, inverse, unit, max, description } = {
     ...defaultGaugeProps,
     ...props,
   };
 
   const asPercentage = fractional ? Math.round(value * max) : value;
   const asActual = max !== 100 ? Math.round(value) : asPercentage;
 
+  const [isHovering, setValue] = useState(false);
+  const handleMouseOver = () => setValue(true);
+  const handleMouseOut = () => setValue(false);
+
+  useEffect(() => {
+    const node = hoverRef.current;
+    if (node) {
```

Comment:
Could be nice to check `isHovering` here as well right?

Reply:
Just confirming, you want to confirm `isHovering` is set?

Reply:
Ah sorry, meant `description`, we only need the listening of that's set right?

Reply:
Ahh okay that makes more sense. Give me a second to update